### ISSUE TITLE
Integration with other libraries (TensorFlow, PyTorch etc)

### ISSUE THREAD

**Issue by sayakpaul** [microsoft/tensorwatch#7]
Kudos to the team for this great library. I was wondering if it is possible to integrate `tensorwatch` with other libraries as mentioned. I am asking this out of my curiosity derived while using `interpret` as it allows for crazy integrations. 

Any example/hint would be great so that I can prepare a notebook and share.

**Comment by sytelus** [comment]
Thank you! TensorWatch currently works well with PyTorch and it should work well with TensorFlow eager mode tensors. We are still working on Keras integration however. BTW, do you mean [this library](https://github.com/Microsoft/interpret) when you mention `interpret`? If you are looking for any specific integrations, we would love to know as well!

**Comment by sayakpaul** [comment]
Yes, I mean that library. 

> TensorWatch currently works well with PyTorch and it should work well with TensorFlow eager mode tensors.

Examples would be great. Also, since `keras` has been merged into `TensorFlow` (from its 1.13 release) I think that advantage can be taken. For example, I used `interpret` to generate explanations from a model created using  `keras` from the `tensorflow` variant by wrapping it into a sklearn compatible version (see [this notebook](https://github.com/sayakpaul/Benchmarking-and-MLI-experiments-on-the-Adult-dataset/blob/master/TF%202.0%20%2B%20interpret.ipynb)).

**Comment by sayakpaul** [comment]
Hey, any updates on it?

**Comment by sytelus** [comment]
Hi @sayakpaul, I've put keras support in our backlog, hope to be done sometime next week. For TensorFlow, we already support eager tensors so no changes there. I would keep this issue open until we are done with keras support.

**Comment by sayakpaul** [comment]
Okay @sytelus. Could you please point me to an example which uses `tensorwatch` and `tensorflow` simultaneously or anything similar?

**Comment by sayakpaul** [comment]
@sytelus a gentle reminder regarding the examples on TF.
